docker/Dockerfile-workers

```diff
@@ -12,7 +12,7 @@ RUN dnf -y install \
     krb5-devel \
     libffi-devel \
     mercurial \
-    npm \
+    nodejs-npm \
```

Comment:
Dnf was acting weird and installing 2 versions of nodejs unless I specified the package as nodejs-npm :man_shrugging: